### DISCUSSION TITLE
Vue-Router use hash mode

### DISCUSSION THREAD

**Discussion by Soya-xy** [hannoeru/vite-plugin-pages#92] [Q&A]
When my Vue-Router use hash mode, the route does not match the path, No match found for location with path "xxxx"

import { createRouter, createWebHashHistory } from "vue-router";
import routes from "virtual:generated-pages";
const router = createRouter({
  history: createWebHashHistory(),
  routes,
});

**Comment by hannoeru** [comment]
Works for me, can you provide a reproduction?

**Reply by Soya-xy** [reply]
Sorry, after I restarted Vite, it worked fine again.

**Comment by ghost** [comment]
When I am trying to build with hash router I am getting this error..
![image](https://user-images.githubusercontent.com/93679925/146922447-5488d46b-4ed8-4f86-a282-48a296f5955a.png)

**Reply by hannoeru** [reply]
Why using hash router with vite-ssg?

**Reply by ghost** [reply]
oh, that's right :/ it doesn't make a sense.
